### PR TITLE
Updated the intro talk slides

### PR DESCRIPTION
**Pull request by agundy**

### CODE REVIEW

File: presentation/index.html [hunk 1]
```diff
@@ -1,371 +1,368 @@
...
+                    </ul>
+                </section>
+
+                <section>
+                    <h2> Who do I talk to for &lt;blank&gt;? </h2>
+                    <img src='https://media.giphy.com/media/si7ZWHm50U3T2/giphy.gif'/>
+                </section>
+
+                <section>
+                    <h2>Spring 2016 Mentors</h2>
```
Comment (seveibar) [inline presentation/index.html]: spring 2016?

File: presentation/index.html [hunk 2]
```diff
@@ -1,371 +1,368 @@
...
+                    </span>
+                    <span class='byside'>
+                        <img width='200' src='img/coordinator_aaron.jpg'/>
+                        <div> Aaron </div>
+                    </span>
+                </section>
+
+
+                <section>
+                    <h2>Returning Spring 2016 Mentors</h2>
```
Comment (seveibar) [inline presentation/index.html]: Spring 2016?

File: presentation/index.html [hunk 3]
```diff
@@ -1,371 +1,368 @@
...
+                    <a href="https://docs.google.com/presentation/d/1Mt_FTDYkzGEN2e515MHgtUq7nZajmomqUgt6DiNExN8/edit?usp=sharing">A Brief History of RCOS</a>
+                    <p class='fragment'>Professor Moorthy</p>
+                </section>
+
+                <section>
+                    <h2> Structure : Members </h2>
+                    <div class='fragment'>
+                        Faculty </br>
+                        <i class='fa fa-arrow-down'></i> <br/>
+                        Coordinators/Mentors <br/>
```
Comment (seveibar) [inline presentation/index.html]: Something should be said about external mentors

File: presentation/index.html [hunk 4]
```diff
@@ -1,371 +1,368 @@
...
+
+                <section>
+                    <h2>Faculty Advisors</h2>
+                    <span class='byside'>
+                        <img width='200' src='img/faculty_moorthy.png'/>
+                        <div> Moorthy </div>
+                    </span>
+                    <span class='byside'>
+                        <img width='200' src='img/faculty_goldschmidt.jpg'/>
+                        <div> Goldschmidt </div>
```
Comment (seveibar) [inline presentation/index.html:78]: is goldschmidt advising this semester?

File: presentation/index.html [hunk 5]
```diff
@@ -1,371 +1,368 @@
...
+                        <li class='fragment'> Mentor </li>
+                        <li class='fragment'> Coordinator </li>
+                        <li class='fragment'> Faculty Member </li>
+                    </ul>
+                </section>
+
+                <section>
+                    <h2> How to be successful (and get an A!) </h2>
+                    <ul>
+                        <li class='fragment'> Commit often (shoot for once a week) </li>
```
Comment (seveibar) [inline presentation/index.html:269]: This is no longer true because of the credit levels

File: presentation/index.html [hunk 6]
```diff
@@ -1,371 +1,368 @@
...
+                        <li class='fragment'> Tell us what you're up to (blog ~3 times a semester) </li>
```
Comment (seveibar) [inline presentation/index.html:270]: Again, this is based on credit levels

**Comment by seveibar** [comment]
where are the history slides?!?

File: presentation/index.html (hunk 2 above)
Comment (agundy) [reply presentation/index.html]: this is mentors that were from Spring 2016 that are returning

File: presentation/index.html (hunk 4 above)
Comment (agundy) [reply presentation/index.html:78]: yes

**Comment by agundy** [comment]
history slides are in the link on the brief history of rcos link

File: presentation/index.html (hunk 5 above)
Comment (agundy) [reply presentation/index.html:269]: what should we add?

Comment (seveibar) [reply presentation/index.html:269]: a link to credit levels seems appropriate

Comment (seveibar) [reply presentation/index.html:269]: and a line indicating that it varies

**Comment by seveibar** [comment]
I thought we were integrating them into the slides?

We're just linking to a google doc?

File: presentation/index.html (hunk 2 above)
Comment (seveibar) [reply presentation/index.html]: I suppose either year makes sense, fair enough :)

File: presentation/index.html (hunk 6 above)
Comment (agundy) [reply presentation/index.html:270]: what should we put in the place here? there is a wide range of credit levels from 1-4.

File: presentation/index.html [hunk 7]
```diff
@@ -1,371 +1,368 @@
...
+                        <li class='fragment'>Unique open-ended learning experience</li>
+                        <li class='fragment'>Gain valuable resume boosters</li>
+                        <li class='fragment'>Huge community of skilled open source developers</li>
+                        <li class='fragment'>Opportunity to work on awesome projects</li>
+                        <li class='fragment'>Credit!?</li>
+                    </ul>
+                </section>
+
+                <section>
+                    <a href="https://docs.google.com/presentation/d/1Mt_FTDYkzGEN2e515MHgtUq7nZajmomqUgt6DiNExN8/edit?usp=sharing">A Brief History of RCOS</a>
```
Comment (seveibar) [inline presentation/index.html:149]: yea this is not supposed to be just a link- i thought we were moving this in

**Comment by agundy** [comment]
that was my understanding

File: presentation/index.html (hunk 7 above)
Comment (seveibar) [inline presentation/index.html:149]: this is a fine last minute solution but it needs to be changed

File: presentation/index.html (hunk 6 above)
Comment (seveibar) [reply presentation/index.html:270]: yea i would indicate that credit levels vary from 1-4 and have different requirements OR just give a link to the grading rubric

Comment (seveibar) [reply presentation/index.html:270]: *AND/OR

File: presentation/index.html (hunk 2 above)
Comment (kmcnellis) [reply presentation/index.html]: Do we need to distinguish new & returning?

File: presentation/index.html (hunk 3 above)
Comment (kmcnellis) [reply presentation/index.html]: 👍

File: presentation/index.html [hunk 8]
```diff
@@ -1,371 +1,368 @@
...
+                        <li class='fragment'><a href='https://github.com/rcos/intro/blob/master/docs/grading.md'>Full Grading Criteria Here</a></li>
+                    </ul>
+                </section>
+
+                <section>
+                    <h1> Goals for RCOS 2016 </h1>
+                </section>
+
+                <section>
+                    <h2> Get more contributions to external projects (projects created outside RCOS) </h2>
```
Comment (kmcnellis) [inline presentation/index.html:281]: This would look better in lowercase?

File: presentation/index.html [hunk 9]
```diff
@@ -1,371 +1,368 @@
...
+                    <h2> Increase longevity of RCOS projects </h2>
+                </section>
+
+                <section>
+                    <h2> Questions? </h2>
+                </section>
+
+                <section>
+                    <h2> What happens next? </h2>
+                    <div class='fragment'> Pitch day! </br><small>Friday for returning projects, Tuesday for new projects</small></div>
```
Comment (kmcnellis) [inline presentation/index.html]: This is inaccurate. Tuesday is small group meeting, next friday is pitches

File: presentation/index.html (hunk 2 above)
Comment (kmcnellis) [reply presentation/index.html]: We have our final list.

File: presentation/index.html [hunk 10]
```diff
@@ -1,371 +1,368 @@
...
+                        <i class='fa fa-arrow-down'></i> <br/>
+                        Members
+                    </div>
+                </section>
+
+                <section>
+                    <h2> Structure : Members </h2>
+                    <div>
+                        Every member gets a mentor <br/>
+                        Mentors are assigned after the first two weeks <br/>
```
Comment (kmcnellis) [inline presentation/index.html]: This is wrong

File: presentation/index.html [hunk 11]
```diff
@@ -1,371 +1,368 @@
...
+                </section>
+
+                <section>
+                    <h2> Who do I talk to for &lt;blank&gt;? </h2>
+                    <img src='https://media.giphy.com/media/si7ZWHm50U3T2/giphy.gif'/>
+                </section>
+
+                <section>
+                    <h2>Fall 2016 Mentors</h2>
+                    <h3>This is where you all come in.</h3>
```
Comment (kmcnellis) [inline presentation/index.html]: No longer accepting applications

File: presentation/index.html (hunk 4 above)
Comment (kmcnellis) [reply presentation/index.html:78]: What about wes?

Comment (agundy) [reply presentation/index.html:78]: oh right... full name somewhere?

Comment (agundy) [reply presentation/index.html:78]: Wes Turner right?

File: presentation/index.html (hunk 8 above)
Comment (agundy) [reply presentation/index.html:281]: which part lowercase?

File: presentation/index.html (hunk 9 above)
Comment (agundy) [reply presentation/index.html]: will update

File: presentation/index.html (hunk 10 above)
Comment (agundy) [reply presentation/index.html]: will remove

File: presentation/index.html (hunk 11 above)
Comment (agundy) [reply presentation/index.html]: learned that this morning. will update

File: presentation/index.html (hunk 8 above)
Comment (kmcnellis) [reply presentation/index.html:281]: H1 is all caps

Comment (agundy) [reply presentation/index.html:281]: oh you mean line 266?

File: presentation/index.html (hunk 4 above)
Comment (agundy) [reply presentation/index.html:78]: Added.

**Comment by agundy** [comment]
Any other comments?

**Pull request merged**